core/shared/src/main/scala/magnolia.scala

```diff
@@ -175,9 +175,32 @@ object Magnolia {
     }
 
     def annotationsOf(symbol: Symbol): List[Tree] = symbol.annotations.collect {
-      case annotation if !(annotation.tree.tpe <:< javaAnnotationType) => annotation.tree
+      case annotation if !(annotation.tree.tpe <:< javaAnnotationType) =>
+        annotation.tree
```

Comment:
We should create a helper method out of this `collect` and use it below as well.

Reply:
I pushed a commit to do that